QuizBowlDiscordScoreTracker/Web/wwwroot/js/main.js

```diff
@@ -1,34 +1,39 @@
 "use strict";
 
-var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
 var nameDisplay = document.getElementById("name");
-var sound1 = document.getElementById("sound1");
-
-var shouldEmitSound = true;
 
 function removeBarrier() {
     document.getElementById("interactBarrier").style.display = "none";
 }
 
-connection.on("PlayerBuzz", function (name) {
-    document.body.style.backgroundColor = "red";
-    nameDisplay.textContent = name;
+if (window.location.search.length <= 1) {
+    nameDisplay.textContent = "No channel specified."
+} else {
+    var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
```

Comment:
Unless you want to support older browsers (like IE 10), you should generally use let and const instead of var, since the scoping rules are saner. If we don't care about supporting IE 10 or before, we should use const for connection and buzzSound, and let for shouldEmitSound.

Reply:
I don't really see any upside to targeting ES6 in this ridiculously simple code -- most of the time when you write ES6 you transpile down to ES5 anyway. None of the variables here are being declared in functions, so `let` doesn't matter, and the code isn't long enough that tracking down reassignment bugs with `const` could save any time at all. I think the however small the benefit of ES5 support is, it's worth it.

Reply:
Fair enough.